pages/common/git-tag.md

```diff
@@ -0,0 +1,19 @@
+# git tag
+
+> Create, list, delete or verify tags
+
+- List all tags
```

Comment:
I never heard of term local used to tags. There can be unpushed tags. There can be not yet downloaded tags. But whenever you work with `git tag` command it always work locally. It is just `git push` and `git fetch/pull` allow you to push and pull your local tags.
`git tag` just show everything it has.